doc/developers/develop.rst

```diff
@@ -521,6 +521,9 @@ poor_score (default=``False``)
     are based on current estimators in sklearn and might be replaced by
     something more systematic.
 
+preserves_32bit_dtype (default=``False``)
```

Comment:
Actually, the common test is a bit broader than only preserving 32 bits. I think that we should go with the tag `preserves_dtype` which should be an array of dtype which will be preserved. By default, it will be only 64 bits

so in short

```suggestion
preserves_dtype (default=``['float64']``)
```

and one could give `['float64', 'float'32', 'float16']`

Reply:
great idea!